### ISSUE TITLE
gazelle moves #keep marker outside of deps, and then removes it next time

### ISSUE THREAD

**Issue by jmuk** [bazelbuild/rules_go#694]
Consider a library in the following deps,
```
go_library(
  name = "go_default_library",
  srcs = [...],
  deps = [
    "//foo", # keep
  ],
)
```

When gazelle processes this, the deps line is tranformed into a single line as
```
deps = ["//foo"],  # keep
```

After the transformation, this `# keep` marker does not mark an entry in deps anymore -- it marks the whole `deps` section. And therefore it has no effect. Another run of gazelle removes this deps completely.

Maybe this reduction to single line looks wrong, gazelle should keep `"//foo",  # keep` line.

**Comment by ianthehat** [comment]
I believe this is the same as #686 which was fixed in #689, could you try with an updated version and see if it works for you now?

**Comment by jmuk** [comment]
Wow, fixed just an hour ago. What a coincidence.
Confirmed fixed.
